.github/workflows/on-release.yaml

```diff
@@ -0,0 +1,27 @@
+name: "Releases"
+
+# This action works when creating a release
+
+on:
+  workflow_dispatch:
+  release:
+    types:
+      - "released"
+
+jobs:
+  tests:
+    uses: ./.github/workflows/tests.yaml
+    
+  publish-production:
+    needs: tests
+    uses: ./.github/workflows/publish.yaml
+    with:
+      image-tag: production
```

Comment:
_If this is defined in any other config, please ignore_
Shouldn't we also specify the `oauth-redirect-url` here?

Reply:
Yes, it's defined, the default value for `oauth-redirect-url` are "https://template-fastapi-react.app.radix.equinor.com".